### DISCUSSION TITLE
리액트 17과 호환되는 버전이 있을까요?

### DISCUSSION THREAD

**Discussion by TaeJunKwon** [zeakd/react-naver-maps#113] [Q&A]
새로운 네이버 맵 도메인  https://oapi.map.naver.com 으로 변경이 된 후 리액트 17과 호환이 되는 버전이 있는지 여쭤보고 싶습니다.

리액트 17로 유지하면서 react-naver-maps 0.1.3을 현재는 이용하기 어려워보여서요

"react": "^17.0.2"
"react-dom": "^17.0.2"
"react-naver-maps": "0.1.3"

**Comment by zeakd** [comment]
어느 부분에서 이슈가 발견되었나요? 현재 react-naver-maps 0.1.3 은 17.0.2 에서 동작합니다.

**Reply by TaeJunKwon** [reply]
`ERROR in ./node_modules/react-naver-maps/dist/chunk-COX7LVBL.mjs 9:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-DI5UAZZG.mjs 9:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-DS7KZOEV.mjs 12:0-56
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
ERROR in ./node_modules/react-naver-maps/dist/chunk-FJVTM3FQ.mjs 8:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-ICJYIRUO.mjs 9:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-IV7GZJYG.mjs 6:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-OJHXJXEG.mjs 9:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-OQI3GZSM.mjs 9:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-PEMPNXAM.mjs 5:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-RNJMH53Z.mjs 4:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-U7EQGDFF.mjs 13:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-VX7BUB5L.mjs 9:0-40
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-XPTCQLXZ.mjs 5:0-46
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#
ERROR in ./node_modules/react-naver-maps/dist/chunk-533MYCRP.mjs 7:0-50
Module not found: Error: Can't resolve 'react/jsx-runtime' in '/Users/front/node_modules/react-naver-maps/dist'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').

![image](https://github.com/zeakd/react-naver-maps/assets/125339740/b471edac-b767-4796-8a45-4206f284e4dc)


jsx-runtime.js 관련된 에러가 발생하고 있습니다.
해당 에러는 이슈쪽에서 한 번 질문했었고 react18로 마이그래이션 시도를 해봤는데 잘되지 않았습니다.
react 17.0.2와 react-naver-maps 0.0.13은 동작이 잘되고 있습니다.

**Reply by TaeJunKwon** [reply]
node_modules/react-naver-maps에 새로운 에러를 발견했는데 관련이 있는지는 잘 모르겠어요
![image](https://github.com/zeakd/react-naver-maps/assets/125339740/290b5452-76b9-489e-919f-73e38b5214b0)
